README.md

```diff
@@ -538,6 +538,37 @@ en:
 Of course, this is just an example. Pundit is agnostic as to how you implement
 your error messaging.
 
+## Deep error messages customisation
```

Comment:
I'm not sure "Deep" feels like a familiar term for me. Maybe something like "Multiple error messages".

Also consider nesting this under the previous section.